tests/llmcompressor/transformers/compression/test_decompress.py

```diff
@@ -0,0 +1,140 @@
+import copy
+import shutil
+import tempfile
+import unittest
+
+import torch
+from compressed_tensors import QUANTIZATION_CONFIG_NAME
+from compressed_tensors.compressors import ModelCompressor
+from compressed_tensors.quantization import QuantizationStatus
+from parameterized import parameterized_class
+from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer
+from transformers.utils.quantization_config import CompressedTensorsConfig
+
+from tests.testing_utils import parse_params, requires_gpu
+
+CONFIG_DIR = "tests/llmcompressor/transformers/compression/decompression_configs"
+
+
+@requires_gpu
+@parameterized_class(parse_params(CONFIG_DIR))
+class TestQuantizationMatches(unittest.TestCase):
+    """
+    Test decompression - given a skeleton model and path to the optimized model,
+    write the optimized model's safetensors to the skeleton model and decompress
+    Ex. write weight_scale to skeleton model and then fp4 -> fp16
+
```

Comment:
this docstring isn't very clear as to what is being tested.

Reply:
Yeah so thats happening. Not sure if you had a chance to look below but the test check thats happening is below here. 27-28. I can put these on top for better visibility